common/main/en.xml

```diff
@@ -7178,6 +7183,11 @@ annotations.
 				<unitPattern count="one">{0} mol</unitPattern>
 				<unitPattern count="other">{0} mol</unitPattern>
 			</unit>
+			<unit type="concentr-item">
+				<displayName>item</displayName>
+				<unitPattern count="one">{0} item</unitPattern>
+				<unitPattern count="other">{0} item</unitPattern>
```

Comment:
doesn't fall back ? I guess we'd need lots of aliases in root for that.

Reply:
There is a general fallback between widths of the items, but en.xml needs to be "fully stocked"